src/components/BoardHeader/HeaderMenu/HeaderMenuItems/HeaderMenuItems.scss

```diff
@@ -0,0 +1,342 @@
+  border: none;
+  border-bottom: 1px dashed $color-white;
+  padding: 0 0 5px 0;
+  outline: 0;
+  transition: border-color 0.1s linear;
+}
+.info__board-name:disabled {
+  border-color: transparent;
+}
+
+.info__access-mode {
+  position: absolute;
+  top: 44px;
+  left: 20px;
+  height: 18px;
+  width: max-content;
+  border: none;
+  background-color: transparent;
+  text-align: left;
+  color: $color-white;
+  opacity: 0.8;
+  font-size: 13px;
+  letter-spacing: 0.27px;
+  line-height: 18px;
+  cursor: pointer;
+  border-bottom: 1px dashed $color-white;
+  padding: 0 0 5px 0;
+  transition: border-color 0.1s linear;
+}
+.info__access-mode:disabled {
+  cursor: default;
+  border-color: transparent;
+}
+.info__access-mode-lock {
+  display: inline-block;
+  position: relative;
+  width: 10px;
+  height: 10px;
+  border: 2px solid $color-white;
+  background-color: $color-white;
+  opacity: 0.8;
+  border-radius: 3px;
+  margin-right: 5px;
+}
+.info__access-mode-lock::before {
+  content: "";
+  position: absolute;
+  height: 6px;
+  width: 6px;
+  background-color: $header-menu-color--light;
+  border-radius: 3px;
+}
+@media (prefers-color-scheme: dark) {
+  .info__access-mode-lock::before {
+    background-color: $header-menu-color--dark;
+  }
+}
+.info__access-mode-lock::after {
+  position: absolute;
+  content: "";
+  width: 6px;
+  height: 6px;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  border: 1px solid $color-white;
+  border-bottom: 0;
+  top: -7px;
+  left: 0px;
+  transition: transform 0.3s linear;
+}
+.info__access-mode-lock--unlocked::after {
+  transform: rotate(-45deg) translate(0, -1px);
+}
+
+.info__edit-button {
+  position: absolute;
+  height: 24px;
+  width: 50px;
+  border: 1.5px solid #f9fafb;
+  border-radius: 32px;
+  background-color: transparent;
+  color: $color-white;
+  cursor: pointer;
+  top: 18px;
+  left: 212px;
+  transition: scale 0.1s ease-in-out;
+}
+.info__edit-button:focus {
+  outline: 0;
+}
+.info__edit-button:active {
+  scale: 0.95;
+}
+
+.header-menu__item {
+  height: $header-menu__item-height;
+  background-color: $color-white;
+  transition: background-color 0.15s ease-in-out;
+  box-sizing: border-box;
+  border-bottom: 1px solid #efefef;
+}
+.header-menu__item:hover {
+  background-color: #f3f3f3;
+}
+@media (prefers-color-scheme: dark) {
+  .header-menu__item {
+    background-color: $color-dark-mode-note;
+  }
+  .header-menu__item:hover {
+    background-color: #3d4555;
+  }
+}
+
+.menu__item-button {
+  background-color: transparent;
+  border: none;
+  padding: none;
+  height: 100%;
+  width: 100%;
+  text-align: left;
+  cursor: pointer;
+}
+.menu__item-button:focus,
+.menu__item-button:hover {
+  outline: none;
+}
+.item-button__icon {
+  width: 79px;
+  height: 24px;
+  color: #a3a6aa;
+  vertical-align: middle;
+}
+
+.item-button__label {
+  height: 24px;
+  width: 180px;
+  color: #111111;
+  font-size: 14px;
+  letter-spacing: 0.5px;
```

Comment:
Please don't use point numbers for `px`. You can't render half a pixel and I don't know how each browser will render such values